config.yml

```diff
@@ -100,7 +100,7 @@ genesis:
             - endpoints:
                 - configs: []
                   rpc_type: JSON_RPC
-                  url: http://anvil:8547
+                  url: http://localhost:8548
```

Comment:
```suggestion
                  # TODO_TECHDEBT(#179): once `relayminer` service is added to tilt, this hostname should point to it instead of `localhost`.
                  url: http://localhost:8548
```